python/setup.py

```diff
@@ -59,16 +61,54 @@ def get_build_version():
     )
     version = os.getenv("RAF_VERSION", default="0.1")
     if not get_env_flag("RELEASE_VERSION", default="0"):
-        version += "+git" + git_sha
+        # version += "+git" + git_sha
+        version = "+git" + git_sha
     return version
 
 
 LIB_LIST = get_lib_path()
 
-# FIXME: commented out _version_ because subprocess.CalledProcessError: Command '['git', 'rev-parse', '--short', 'HEAD']' returned non-zero exit status 128.
 # __version__ = get_build_version()
-
-__version__ = "0.1"
+git_version = get_build_version()
+
+
+def get_build_raf_version():
+    raf_build_version = os.getenv("RAF_BUILD_VERSION")
+    raf_build_platform = os.getenv("RAF_BUILD_PLATFORM")
+    version_file = open("./raf/version.txt", "r")
```

Comment:
1. This path won't work if you run `python3 setup.py` under `raf/python`.
2. Use `with open(...)` if you want to open and read files.